### ISSUE TITLE
Include References to Modular Docs

### ISSUE THREAD

**Issue by nengard** [redhat-documentation/community-collaboration-guide#90]
To connect our efforts we should have a section that references the modular docs initiative content and guidelines. https://redhat-documentation.github.io/modular-docs/

**Comment by pmkovar** [comment]
Yep, something to add to checklists as a consideration before establishing an upstream-downstream workflow. Mostly related to migrations (#63).
